solr/core/src/test-files/solr/configsets/cloud-minimal-faulty/conf/schema.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Maybe there is a way to not have to create this new configset? either use the Config API, or the configset API, or maybe just updating one file? or maybe set a system property that one of the existing configsets will use to something invalid?